src/otg/common/Liftover.py

```diff
@@ -38,8 +40,17 @@ def __init__(
         self.chain_file = chain_file
         self.max_difference = max_difference
 
-        # Initializing liftover object by opening the chain file:
-        self.lo = LiftOver(chain_file)
+        # Initializing liftover object by opening the chain file - LiftOver only supports local files:
+        if chain_file.startswith("gs://"):
+            bucket_name = chain_file.split("/")[2]
+            blob_name = "/".join(chain_file.split("/")[3:])
+            with tempfile.NamedTemporaryFile(delete=True) as temp_file:
+                storage.Client().bucket(bucket_name).blob(
+                    blob_name
+                ).download_to_filename(temp_file.name)
+                self.lo = LiftOver(temp_file.name)
+        else:
+            self.lo = LiftOver(chain_file)
 
```

Comment:
I'm wondering if it worth abstracting this functionality into the utils methods. OK, at this point this is the only application that cannot handle google storage, however I see this might be an issue in the future.

Also not 100% sure how this solution would be portable for aws and azure storages. (Just thinking about @addramir's promise to make the pipeline portable.) It's not a blocker in any ways, just thinking about design/portability.

Reply:
I also thought about this and agree with you. This code would crash if the path is pointing to a different cloud provider. 
I decided not to do anything about it because the former solution of using gcsfs would lead to the same results, and because in general we're quite locked in Google's cloud platform. Porting this to other cloud providers will have more ramifications. So far, this solution works with local files and with GCS files.